nilearn/_utils/testing.py

```diff
@@ -647,3 +649,20 @@ def assert_less_equal(a, b):
     def assert_less(a, b):
         if a >= b:
             raise AssertionError("%f is not less than %f" % (a, b))
+
+
+def MemoryLimit(limit):
```

Comment:
I use it as a decorator afterward. But I can change it to `snake_case` since nose does that with `with_setup`.